tests/kola/test-bootupd

```diff
@@ -63,7 +63,7 @@ prepare_efi_update
 # echo somenewfile > ${ostefi}/somenew.efi
 rm -v ${ostefi}/shim.efi
 echo bootupd-test-changes >> ${ostefi}/grubx64.efi
-bootupctl backend generate-update-metadata /
+bootupd backend generate-update-metadata /
```

Comment:
This will also need to be `/usr/libexec/bootupd` since `/usr/libexec` is intentionally not in `$PATH`.